gpkit/model.py

```diff
@@ -544,6 +544,30 @@ def _latex(self, unused=None):
                                  for var, val in self.constants.items()]) +
                          ["\\end{array}"])
 
+    def interact(self,  fn_of_sol=None, ranges=None, **solveargs):
```

Comment:
I'm ok with the name; would `widget` be another option?

This needs a docstring now that it's part of `Model`, in my opinion.